network-api/networkapi/mozfest/templates/partials/streamfield.html

```diff
@@ -1,9 +1,15 @@
 {% load wagtailcore_tags %}
 
 {% for block in page.body %}
-  {% if block.block_type == 'heading' %}
-    <h1>{{ block.value }}</h1>
-  {% else %}
-    {% include_block block %}
-  {% endif %}
+{% if block.block_type == 'paragraph'%}
+<div class="container {{block.block_type}}-block">
+  <div class="row justify-content-center">
+    <div class="streamfield-content">
+      {% include_block block with parent_page=page page_type="blog"%}
```

Comment:
Why this page_type? mozfest pages aren't blog pages?

Reply:
Could be the same style of html element nesting as the blog? (Im guessing). If that's true that, let's add a Django template comment to state what its "blog" and not "mozfest".

Reply:
Hi Everyone, sorry about that! I left those there by mistake, and have removed them in the latest commit. Thanks!